.gitignore

```diff
@@ -9,6 +9,7 @@
 /ext/gpac
 /ext/libargparse
 /ext/libjpeg
+/ext/libjpeg-turbo
```

Comment:
Should we delete the `/ext/libjpeg` line?

Reply:
I thought I could leave it for now. If users update their local repositories instead of cloning fresh ones, it will avoid an eventual `libjpeg` entry popping into `git status`.

Reply:
Seeing the `ext/libjpeg` entry in the `git status` output could remind the user to delete the directory. That seems good.

Reply:
> Seeing the `ext/libjpeg` entry in the `git status` output could remind the user to delete the directory.

Good point.  
The case I had in mind was switching between the `main` and `v1.0.x` branches while keeping both `ext/libjpeg` and `ext/libjpeg-turbo`.

> That seems good.

Not sure what this refers to.

Reply:
I meant it seems good to have the `ext/libjpeg` entry in the `git status` output, because it could remind the user to delete the directory.